tests/gpflow/utilities/test_utilities.py

```diff
@@ -4,12 +4,52 @@
 import tensorflow_probability as tfp
 
 import gpflow
+from gpflow import Parameter
 from gpflow.config import Config, as_context
 from gpflow.models.util import data_input_to_tensor
-from gpflow.utilities import positive, triangular
+from gpflow.utilities import positive, triangular, set_trainable
 from gpflow.utilities.ops import difference_matrix
 
 
+def _module() -> tf.Module:
```

Comment:
the `utilities` structure is too packed as it is - if you're already moving tests & adding more, how about putting them in a `test_set_trainable` file in this directory ?

Reply:
doesn't that make it harder to find tests, if they don't follow the same directory structure as the code?

Reply:
I think a file called "test_set_trainable" makes them easier to find than having to traverse through a long file (note that e.g. test_deepcopy is already separate). The key issue is that gpflow.utilities.utilities has too much stuff and should be split - but that's more effort and probably better-suited for a separate PR.

Reply:
done. btw there are only 8 tests in the test_utilities module